src/OptPass.cc

```diff
@@ -30,14 +30,26 @@
  * SUCH DAMAGE.
  */
 
-#include "PolicyFile.hh"
 
 #include "llvm/Pass.h"
+
+#include "llvm/IR/Module.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/CallingConv.h"
+#include "llvm/IR/IRBuilder.h"
+#include "llvm/IR/InstIterator.h"
+#include "llvm/IR/TypeBuilder.h"
 #include "llvm/Support/raw_ostream.h"
+#include <llvm-c/Core.h>
+#include "PolicyFile.hh"
+
 
 using namespace llvm;
 using namespace loom;
-
+Function* createInstrFunction(Module &module, CallInst* callInst, Policy::Direction direction, Policy& policy);
```

Comment:
These helper functions should either be static or declared inside the anonymous namespace (with `OptPass`).